compose/cli/formatter.py

```diff
@@ -23,3 +24,7 @@ def table(self, headers, rows):
         table.set_chars(['-', '|', '+', '-'])
 
         return table.draw()
+
+    def json(self, headers, rows):
+        rows.insert(0, headers)
+        return json.dumps(rows)
```

Comment:
I don't think this should be part of `Formatter()`. I see json output as a separate alternative to this human readable formatting.  Let's just call `json.dumps()` directly from `main`.